tinyms/vision/transforms.py

```diff
@@ -12,10 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ============================================================================
-
 import numpy as np
-import tinyms as ts
 from PIL import Image
+import tinyms as ts
+from tinyms import vision
```

Comment:
Please use `from . import TypeCast` instead.

Reply:
Thanks.I have corrected the codes.